common/utilities/integration_adaptors_logger.py

```diff
@@ -144,15 +67,43 @@ def _formatted_string(self, message: str, dict_values: dict) -> str:
         formatted_values = self._format_values_in_map(dict_values)
         return message.format(**formatted_values)
 
-    @staticmethod
-    def _format_exception(message: str) -> str:
-        """
-        Returns a formatted string of the last exception on the stack
-        """
-        exc_inf = sys.exc_info()
-        if all(exc_inf):
-            (etype, value, trace) = exc_inf
-            if message:
-                return f'{message}, {traceback.format_exception(etype, value, trace)}'
-            else:
-                return f'{traceback.format_exception(etype, value, trace)}'
+
+class CustomFormatter(logging.Formatter):
+    def __init__(self):
+        super().__init__(
+            fmt='%(asctime)sZ | %(levelname)s | %(process)d | %(interaction_id)s | %(message_id)s | %(correlation_id)s | %(inbound_message_id)s | %(name)s | %(message)s',
+            datefmt='%Y-%m-%dT%H:%M:%S.%f')
+
+    def format(self, record: LogRecord) -> str:
+        record.message_id = message_id.get() or ''
```

Comment:
You should be able to use `record.message_id = message_id.get(default='')` or just specify default when each context variable is created (top of this file)  as doc on that method specifies:

> If there is no value for the variable in the current context, the method will:
         * return the value of the default argument of the method, if provided; or
         * return the default value for the context variable, if it was created
           with one; or
         * raise a LookupError.